test/unit/commands/doctor/startup-spec.js

```diff
@@ -8,157 +8,342 @@ const setupEnv = require('../../../utils/env');
 
 const modulePath = '../../../../lib/commands/doctor/checks/startup';
 
-describe('Unit: Commands > Startup Checks', function () {
-    const sandbox = sinon.sandbox.create();
-
-    afterEach(function () {
-        sandbox.restore();
-    });
-
-    it('exports one task', function () {
+describe('Unit: Doctor Checks > Startup', function () {
+    it('exports tasks', function () {
         const startupChecks = require(modulePath);
 
         expect(startupChecks).to.be.an.instanceof(Array);
-        expect(startupChecks).to.have.length(1);
+        expect(startupChecks).to.have.length(4);
         expect(startupChecks[0].title).to.match(/Validating config/);
         expect(startupChecks[0].task).to.be.an.instanceof(Function);
+        expect(startupChecks[1].title).to.match(/Install folder permissions/);
+        expect(startupChecks[1].task).to.be.an.instanceof(Function);
+        expect(startupChecks[2].title).to.match(/Install directory file permissions/);
+        expect(startupChecks[2].task).to.be.an.instanceof(Function);
+        expect(startupChecks[3].title).to.match(/Content folder owner/);
+        expect(startupChecks[3].task).to.be.an.instanceof(Function);
     });
 
-    it('rejects if environment is passed and no config exists for that environment', function () {
-        const env = setupEnv();
-        const cwdStub = sandbox.stub(process, 'cwd').returns(env.dir);
-        const startupChecks = require(modulePath);
+    describe('Config validation', function () {
+        const sandbox = sinon.sandbox.create();
+
+        afterEach(function () {
+            sandbox.restore();
+        });
 
-        return startupChecks[0].task({environment: 'testing'}).then(() => {
-            env.cleanup();
-            expect(false, 'error should have been thrown').to.be.true;
-        }).catch((error) => {
-            expect(error).to.be.an.instanceof(errors.ConfigError);
-            expect(error.message).to.match(/Config file is not valid JSON/);
-            expect(error.options.environment).to.equal('testing');
-            expect(cwdStub.calledOnce).to.be.true;
+        it('rejects if environment is passed and no config exists for that environment', function () {
+            const env = setupEnv();
+            const cwdStub = sandbox.stub(process, 'cwd').returns(env.dir);
+            const configValidationCheck = require(modulePath);
 
-            env.cleanup();
+            return configValidationCheck[0].task({environment: 'testing'}).then(() => {
+                env.cleanup();
+                expect(false, 'error should have been thrown').to.be.true;
+            }).catch((error) => {
+                expect(error).to.be.an.instanceof(errors.ConfigError);
+                expect(error.message).to.match(/Config file is not valid JSON/);
+                expect(error.options.environment).to.equal('testing');
+                expect(cwdStub.calledOnce).to.be.true;
+
+                env.cleanup();
```

Comment:
This isn't related specifically to the PR, but we should probably move this to the top of the block - if any of the assertions fail, this will never be called